### PR TITLE
[zh-cn]: Update translation of String.length

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#13975

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/string/length/index.md [hunk 1]
```diff
@@ -5,32 +5,35 @@ slug: Web/JavaScript/Reference/Global_Objects/String/length
...
+## 值
+
+一个非负整数。
+
+{{js_property_attributes(0, 0, 0)}}
+
 ## 描述
 
-该属性返回字符串中的代码单元数量。JavaScript 使用 [UTF-16](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String#utf-16_字符、unicode_码位和字素簇（grapheme_clusters）) 编码，其中每个 Unicode 字符可以编码为一个或两个代码单元，因此 `length` 返回的值可能与字符串中 Unicode 字符的实际数量不匹配。对于拉丁文、西里尔文、众所周知的 CJK 字符等常见脚本，这应该不是问题，但如果你正在使用某些脚本，例如表情符号、[数学符号](https://zh.wikipedia.org/wiki/数学字母数字符号)或晦涩的汉字，你可能需要考虑代码单元和字符之间的差异。
+该属性返回字符串中的码元数量。JavaScript 使用 [UTF-16](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String#utf-16_字符、unicode_码位和字素簇) 编码，其中每个 Unicode 字符可以编码为一个或两个码元，因此 `length` 返回的值可能与字符串中 Unicode 字符的实际数量不匹配。对于拉丁文、西里尔文、众所周知的 CJK 字符等常见脚本，这应该不是问题，但如果你正在处理某些脚本，例如表情符号、[数学符号](https://zh.wikipedia.org/wiki/数学字母数字符号)或生僻字，你可能需要考虑码元和字符之间的差异。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/string/length/index.md:21]: https://en.wikipedia.org/wiki/Byte#Multiple-byte_units

注意这里的单位

File: files/zh-cn/web/javascript/reference/global_objects/string/length/index.md [hunk 2]
```diff
@@ -5,32 +5,35 @@ slug: Web/JavaScript/Reference/Global_Objects/String/length
...
 
-语言规范要求字符串的最大长度为 2<sup>53</sup> - 1 个元素，这是[精确整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)的上限。但是，具有此长度的字符串需要 16384TB 的存储空间，这在任何合理的设备内存中都容纳不了，因此实现倾向于降低阈值，从而允许字符串的长度方便地存储在 32 位整数中。
+语言规范要求字符串的最大长度为 2<sup>53</sup> - 1 个元素，这是[精确整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)的上限。但是，具有此长度的字符串需要 16384TB 的存储空间，这远远超出了任何合理设备的内存限制，因此实现通常会降低这个阈值，从而允许字符串的长度方便地存储在一个 32 位整数中。
 
-- 在 V8（Chrome 和 Node 使用）中，最大长度为 2<sup>29</sup> - 24（\~1GB）。在 32 位系统上，最大长度为 2<sup>28</sup> - 16（\~512MB）。
-- 在 Firefox 中，最大长度为 2<sup>30</sup> - 2（\~2GB）。在 Firefox 65 之前，最大长度为 2<sup>28</sup> - 1（\~512MB）。
-- 在 Safari 中，最大长度为 2<sup>31</sup> - 1（\~4GB）。
+- 在 V8 引擎（Chrome 和 Node 使用）中，最大长度为 2<sup>29</sup> - 24（约为 1GB）。在 32 位系统上，最大长度为 2<sup>28</sup> - 16（约为 512MB）。
+- 在 Firefox 中，最大长度为 2<sup>30</sup> - 2（约为 2GB）。在 Firefox 65 之前，最大长度为 2<sup>28</sup> - 1（约为 512MB）。
+- 在 Safari 中，最大长度为 2<sup>31</sup> - 1（约为 4GB）。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/string/length/index.md:26]: https://en.wikipedia.org/wiki/Byte#Multiple-byte_units

注意单位

Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/string/length/index.md:26]:
```suggestion
- 在 V8 引擎（Chrome 和 Node 使用）中，最大长度为 2<sup>29</sup> - 24（约为 1GiB）。在 32 位系统上，最大长度为 2<sup>28</sup> - 16（约为 512MiB）。
- 在 Firefox 中，最大长度为 2<sup>30</sup> - 2（约为 2GiB）。在 Firefox 65 之前，最大长度为 2<sup>28</sup> - 1（约为 512MiB）。
- 在 Safari 中，最大长度为 2<sup>31</sup> - 1（约为 4GiB）。
```

File: files/zh-cn/web/javascript/reference/global_objects/string/length/index.md [hunk 3]
```diff
@@ -5,32 +5,35 @@ slug: Web/JavaScript/Reference/Global_Objects/String/length
...
+
+{{js_property_attributes(0, 0, 0)}}
+
 ## 描述
 
-该属性返回字符串中的代码单元数量。JavaScript 使用 [UTF-16](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String#utf-16_字符、unicode_码位和字素簇（grapheme_clusters）) 编码，其中每个 Unicode 字符可以编码为一个或两个代码单元，因此 `length` 返回的值可能与字符串中 Unicode 字符的实际数量不匹配。对于拉丁文、西里尔文、众所周知的 CJK 字符等常见脚本，这应该不是问题，但如果你正在使用某些脚本，例如表情符号、[数学符号](https://zh.wikipedia.org/wiki/数学字母数字符号)或晦涩的汉字，你可能需要考虑代码单元和字符之间的差异。
+该属性返回字符串中的码元数量。JavaScript 使用 [UTF-16](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String#utf-16_字符、unicode_码位和字素簇) 编码，其中每个 Unicode 字符可以编码为一个或两个码元，因此 `length` 返回的值可能与字符串中 Unicode 字符的实际数量不匹配。对于拉丁文、西里尔文、众所周知的 CJK 字符等常见脚本，这应该不是问题，但如果你正在处理某些脚本，例如表情符号、[数学符号](https://zh.wikipedia.org/wiki/数学字母数字符号)或生僻字，你可能需要考虑码元和字符之间的差异。
 
-语言规范要求字符串的最大长度为 2<sup>53</sup> - 1 个元素，这是[精确整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)的上限。但是，具有此长度的字符串需要 16384TB 的存储空间，这在任何合理的设备内存中都容纳不了，因此实现倾向于降低阈值，从而允许字符串的长度方便地存储在 32 位整数中。
+语言规范要求字符串的最大长度为 2<sup>53</sup> - 1 个元素，这是[精确整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)的上限。但是，具有此长度的字符串需要 16384TB 的存储空间，这远远超出了任何合理设备的内存限制，因此实现通常会降低这个阈值，从而允许字符串的长度方便地存储在一个 32 位整数中。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/string/length/index.md:22]:
```suggestion
语言规范要求字符串的最大长度为 2<sup>53</sup> - 1 个元素，这是[精确整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)的上限。但是，具有此长度的字符串需要 16384TiB 的存储空间，这远远超出了任何合理设备的内存限制，因此实现通常会降低这个阈值，从而允许字符串的长度方便地存储在一个 32 位整数中。
```

**Review by jasonren0403 — APPROVED** [review]
Almost good 👍

**Pull request merged**